firmware/lucidgloves-firmware/src/Util/DataStructs.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+#include "../../AdvancedConfig.h"
+
+const char* const SPECIAL_COMMANDS[] = {
+    "SaveInter",
+    "SaveTravel",
+    "ClearData"
+    // Add more commands as needed
+};
+
+//NUM_SPECIAL_COMMANDS can be defined this way because all char* are pointers of the same size
+const int NUM_SPECIAL_COMMANDS = sizeof(SPECIAL_COMMANDS) / sizeof(SPECIAL_COMMANDS[0]);
```

Comment:
Prefer an enum, and perhaps a `std::map<ESpecialCommands, String>` if you do need to link them back to strings.

```suggestion
enum ESpecialCommands {
    SPECIAL_COMMAND_SAVE_INTER,
    SPECIAL_COMMAND_SAVE_TRAVEL,
    SPECIAL_COMMAND_CLEAR_DATA,
    NUM_SPECIAL_COMMANDS
}
```